internal/cmd/base/servers.go

```diff
@@ -223,6 +223,16 @@ func (b *Server) PrintInfo(ui cli.Ui) {
 		b.Info["cgo"] = "enabled"
 	}
 
+	var (
+		tokDurKey = "auth token max duration"
+		tokStaKey = "auth token max staleness"
+	)
+
+	b.InfoKeys = append(b.InfoKeys, tokDurKey)
+	b.Info[tokDurKey] = fmt.Sprintf("%gs", globals.DefaultAuthTokenMaxDuration.Seconds())
```

Comment:
Please use `%ds`, or use `>String()` instead if you want the Go-style duration string.

Reply:
Fixed.